### ISSUE TITLE
ISRM (ACCST/ACCESS) & Receivers v2.1.0 firmware download and test

### ISSUE THREAD

**Issue by Adelaxie** [FrSkyRC/Firmware-Test#19]
### All firmware can be downloaded from the Code section.

The firmware v2.1.0 is to provide updates for ISRM and ACCESS receivers.
It will work with v2.1.0 ACCST receivers and ACCESS receivers with updates made simultaneously.
If a new version updated in the near future, the version number will be  v2.1.x A, B, C .... as differences.
Please reply to the post or create a new issue post with found issues after making sure the latest firmware is updated in this post.
Thanks!
1. ISRM-P for X-Lite Pro, ISRM-S for X9DP 2019/X9DP 2019 SE
PLACE HOLDER FOR NOW

2. Receiver firmware for RX6R, RX4R
PLACE HOLDER FOR NOW

**Comment by Adelaxie** [comment]
PLACE HOLDER

**Comment by Adelaxie** [comment]
PLACE HOLDER

**Comment by Hennyvt** [comment]
Hi Adela. Does the place holder mean the ACCESS firmware will be available soon. Am really keen to test it in my new X10S Express radio.

Thank you henny

**Comment by Adelaxie** [comment]
yes, It will start from today and fix the issues on unreleased ACCST receivers at the same time.

**Comment by Hennyvt** [comment]
Hi Adela.  I am so sad, still no ISRM for my HORUS EXPRESS. Are you able to give me an approx time for this release.
Waiting in anticipation Henny.

**Comment by Adelaxie** [comment]
@Hennyvt If the first batch tested good without critical issues I will keep updating other firmwares.But if you would like to participate in alpha test about this X10 Express upgrade, write a mail to me (libo.xie@frsky-rc.com)so that I can share it with you .

**Comment by itsmillertime0** [comment]
Is this functional? I updated Taranis 2019 + SE to OpenTX 2.3.7, flashed internal module with firmware downloaded here, and flashed both a rx6r and s8r with 2.1.0 but neither will bind in access or accst.

**Comment by franz4949** [comment]
Hi Adela, OpenTx version 2.3.7 is out, dated 28th March, thus maybe you edit your hint for nightly builds in your starting post and advise for the latest OTx version instead.

**Comment by proublot** [comment]
Hi Adela, have you a modify Tool-FrSky update sport : .frsk files cannot be recognised

**Comment by ReinhardZ1** [comment]
Flashed my X9D+SE 2019 with ISRM2.1.0.A
Binding to receiver X6R with SW 2.1.0 in 8ch mode: no problem
- no FL sensor in ACCST D16 mode
- FL bits on SBus are filtered (hope this will change in future)

**Comment by ReinhardZ1** [comment]
> Hi Adela, have you a modify Tool-FrSky update sport : .frsk files cannot be recognised

Rename the file to *.frk, then STK tool will load

**Comment by ReinhardZ1** [comment]
ACCESS SW 2.1.0A for receiver RX6R:
Receiver cannot be registered with transmitter running OTX 2.3.5
(readme points out, that OTX version later than 24th March has to be used)

**Comment by Hennyvt** [comment]
ACCESS SW 2.1.0A  for receiver RX6R: And Alpha firmware for HORUS EXPRESS 2.1A I am running OTX2.3.7 and am unable to register the RX6R as well.

**Comment by alm-nl** [comment]
New files for the trasnmitters were silently uploaded:
ISRM-P for X-Lite Pro, ISRM-S for X-Lite S
FW-XLiteSPro-ISRM-v2.1.0**B**.zip

ISRM-S for X9DP 2019/X9DP SE 2019
FW-X9DP2019-ISRM-v2.1.0**B**.zip

See first message for the links.
Maybe try those?

**Comment by ReinhardZ1** [comment]
FW-X9DP2019-ISRM-v2.1.0B uploaded to X9DP2019 transmitter (OTX2.3.5):
Version menu shows: ISRM-S-X10E   1.10/2.1.0EU

Is this the correct ISRM FW for X9DP2019 ?
![ISRM_v2 1 0B](https://user-images.githubusercontent.com/62476600/77911893-df46b980-7291-11ea-9da4-838b778ee8c9.jpg)

**Comment by EngelMT** [comment]
You are right. There must be an issue. Will inform Adela about it. Tested in the moment with my own X9D Plus 2019.

**Comment by Adelaxie** [comment]
> New files for the transmitters were silently uploaded:
> ISRM-P for X-Lite Pro, ISRM-S for X-Lite S
> FW-XLiteSPro-ISRM-v2.1.0**B**.zip
> 
> ISRM-S for X9DP 2019/X9DP SE 2019
> FW-X9DP2019-ISRM-v2.1.0**B**.zip
> 
> See first message for the links.
> Maybe try those?

Please check the code section for the update logs to know if there is any firmware updates.
thanks

**Comment by Adelaxie** [comment]
> Is this functional? I updated Taranis 2019 + SE to OpenTX 2.3.7, flashed internal module with firmware downloaded here, and flashed both a rx6r and s8r with 2.1.0 but neither will bind in access or accst.

Please check the hardware page about your firmware mode (FCC or EU).

**Comment by Adelaxie** [comment]
> Hi Adela, have you a modify Tool-FrSky update sport : .frsk files cannot be recognised

Will provide new version in FrSky website soon.

**Comment by crazybiker84** [comment]
> Hi Adela, have you a modify Tool-FrSky update sport : .frsk files cannot be recognised

If you change the suffix to *.frk, the tool will load it.

**Comment by Hennyvt** [comment]
Regarding the X10S Express with ISRM Alpha firmware ISRM-S_X10E-2.1.0A I then flashed the RX6R with version ACCST and ACCESS version 2.1.0 but I was unable to get the receiver to bind in either protocol. So Today I flashed the receiver using my X9D+(Old Version) and success. So there is something in the X10S Express radio being unable to flash this version of receiver firmware. I tried an earlier version from my X10S and this completes the flashing fine.

**Comment by Hennyvt** [comment]
Regarding the X10S Express with ISRM Alpha firmware ISRM-S_X10E-2.1.0A I then flashed the RX6R using the X10S EXPRESS with version ACCST and ACCESS version 2.1.0 but I was unable to get the receiver to bind in either protocol. So Today I flashed the receiver using my X9D+(Old Version) and success flashing the RX6R with 2.1A. So there is something in the X10S Express radio being unable to flash this version of receiver firmware. I tried an earlier version from my X10S and this completes the flashing fine.

**Comment by Adelaxie** [comment]
@Hennyvt Given no X10 express firmware updated here, I think its better not to reply here causing confuses. thanks.

**Comment by ReinhardZ1** [comment]
> > Hi Adela, have you a modify Tool-FrSky update sport : .frsk files cannot be recognised
> 
> If you change the suffix to *.frk, the tool will load it.

Yes, the tool will load it, but you cannot register the receiver.
If you flash the receiver via the transmitter (X9DPSE2019 in my case) with the *.frk file, you can register and bind the receiver (tested witf RX6R , 2.1.0A SW). Transmitter was running OTX 2.3.7

**Comment by ReinhardZ1** [comment]
> You are right. There must be an issue. Will inform Adela about it. Tested in the moment with my own X9D Plus 2019.

Is there any reaction?

**Comment by EngelMT** [comment]
FrSky told me that it is only a bug in the name. Will be fixed with next update.

**Comment by alexdibi** [comment]
since many have some receivers that cannot be updated to the new protocol having to disassemble everything, to avoid that every time you need to switch from an accst v1 firmware to an accst v2 one, it would not be appropriate to include them both in the firmware, perhaps with a "warning"  in front of the D16 v1?

**Comment by ParkerEde** [comment]
Hi. I am still waiting for taranis q x7s access and horus x10 isrm upgrade module firmware. When will it go an with it? Kind reagards

**Comment by ReinhardZ1** [comment]
Today I've made a test flight with an electric glider. RC equipment used:
Taranis X9DP SE 2019 running OTX 2.3.7, ISRM v2.1.0B (LBT)
RX6R with FW ACCESS 2.1.0A
Connected telemetry sensors:
1.) OpenXsensor Vario/GPS/Voltage
2.) Arduino based Link Quality sensor (value 5103). 5103 estimates the true FL-Bit as it was sent on SBus with V1 software.

Result: 
- No spikes in telemetry data (spikes in telemetry data occur with ACCST SW 2.1.0) , good!
- FLR% shows worse link quality than 5103 value. As we don't have the unfiltered FL-Bit on SBus, it's hard to say, which LQ value is correct.

Flight log can be downloaded from [RCG thread](https://www.rcgroups.com/forums/showpost.php?p=44213049&postcount=2297)

First picture shows telemetry data with ACCESS SW (no spikes)
Second picture shows telemetry data with ACCST SW from former flight (with spikes)
Third picture shows comparison of 5103 value and FLR%, respectively calculated FLR100 = 100-FLR%
![Eclipse_Flight3_3](https://user-images.githubusercontent.com/62476600/78815344-6d2c4e00-79d0-11ea-9b30-661b0acd5bf7.jpg)


![Eclipse_Flight3_4](https://user-images.githubusercontent.com/62476600/78815167-263e5880-79d0-11ea-9e72-57c3e21ea50c.jpg)
![Eclipse_Flight2_3](https://user-images.githubusercontent.com/62476600/78815194-322a1a80-79d0-11ea-94f4-61db1a1b4f89.jpg)

**Comment by afdofdez** [comment]
Hi Adela, when will the new version of the ISRM be available for horus x10? 
Thank you.

**Comment by Burgerman-X10express** [comment]
I too would like to know this. 
And would love the latest beta or alpha version too. Please send to burgerman@ntlworld.com

**Comment by ReinhardZ1** [comment]
After installing ISRM FW 2.1.0J on the Taranis X9DP SE 2019 (LBT, OTX 2.3.7), I get immediately the message 'TX antenna problem!' and the voice alarm 'radio antenna defective'. The same happens, if you install V1 ISRM SW 1.1.3. I think this problem is related to OTX 2.3.7.
With ISRM 2.1.0B this does not happen.
![AntennaMessage](https://user-images.githubusercontent.com/62476600/79039125-61fb3d00-7bdf-11ea-96f3-e4918bc95f2e.jpg)

**Comment by Adelaxie** [comment]
@ReinhardZ1 
Could you please confirm if the radio has the normal range(to confirm the output power is good)? We can not repeat it but I highly suspect there is something wrong with X9DP SE 2019  as I have several SWR reports of this radio as well.

**Comment by ReinhardZ1** [comment]
> @ReinhardZ1
> Could you please confirm if the radio has the normal range(to confirm the output power is good)? We can not repeat it but I highly suspect there is something wrong with X9DP SE 2019 as I have several SWR reports of this radio as well.

But why does this not happen with ISRM FW 2.1.0B ?

I've made a flight with an electric glider with this transmitter with ISRM 2.1.0B and receiver RX6R with ACCESS SW 2.1.0A. Attached is a log of the flight. Distance was up to 400m
![Eclipse_RX6R_ACCESS_2 1 0A](https://user-images.githubusercontent.com/62476600/79318310-17433300-7f07-11ea-8856-8722f083e841.jpg)
[Eclipse-2020-04-11-Session1.csv.txt](https://github.com/FrSkyRC/Firmware-Test/files/4480074/Eclipse-2020-04-11-Session1.csv.txt)

**Comment by TschorschR** [comment]
Hi - it's me again.

My Tx is a Taranis X9D running Open TX 2.3.7, tested the *E version for G-RX8.
Still sensor lost and frozen display.

Video (see Sensor Lost and Frozen Vsp.zip)/pics attached.

Georg

![IMG_20200415_125034](https://user-images.githubusercontent.com/60703945/79334247-31d5d600-7f20-11ea-8220-726a7079c886.jpg)
![IMG_20200415_124947](https://user-images.githubusercontent.com/60703945/79334248-326e6c80-7f20-11ea-99e1-196781ecf6d2.jpg)
[Sensor Lost and Frozen Vsp.zip](https://github.com/FrSkyRC/Firmware-Test/files/4480998/Sensor.Lost.and.Frozen.Vsp.zip)

**Comment by ParkerEde** [comment]
Hi,
now I've flashed my Taranis Q X7S ACCESS ISRM module to 2.1.0J and also one RX4R receiver to 2.1.0J .
After flashing both, I've   re REGISTER and re BIND the RX4R successfully.

But now there is a new problem. When I powered on the Lipo to ESC, the servos make a big movement and then there comes a second spike from the receiver and the servos pull out again. After that, the servos go to their normal position. Here you can see this issue:

https://drive.google.com/file/d/1WKXaeeiSZhYfa3rf-3MSnuqtDLniogMa/view?usp=sharing

![IMG_4184](https://user-images.githubusercontent.com/15068446/79356559-6a38dc80-7f3f-11ea-8392-36603b2d4db7.jpg)

I know that some servos do the first movement, when powered on. But the second movement is new with this firmware and not okay.

Kind regards
Ralf

**Comment by Burgerman-X10express** [comment]
Just added the ISRM firmware beta test. Or alpha?

 

Radio Horus x10 Express.

Firmware is the horus x10 express, with a j at the end.

Here https://github.com/FrSkyRC/Firmware-Test/issues/19

 

It almost but not quite bricks the module. The FROS operating system cannot see a module, and says its turned off. It also says the R9M 2019 external is off too. I turned internal and external on/off in software many times. No change. There is zero RF output on both/either alone. And the MONITOR does not move when the sticks do.

I reloaded fros, and tried to reflash internal module many times. After a few secs it says its failed. Almost ready to give up and I was able to flash the internal module with the old 1.1.03  ? or whatever it is.  Only possible by turning off internal and external modules. Now it works again.

 

Any ideas?

**Comment by Burgerman-X10express** [comment]
Just installed ISRM-S-X10E for X10/X10S Express - FW-X10X10SEXPRESS-ISRM-v2.1.0J.zip
On to a brand new and up to date, X10 Express. Twice in case its something I did.
It has the 900Mhz M9 2019 module in the rear bay. On latest 1.3 Access NonEU. 
Provided both modules are turned OFF and the TX rebooted, the sticks mone the various channels on the Monitor screen. And theres no RF out. (I have a signal strength meter).
The moment the INTERNAL 2.4 ISRM is turned on, or the Module bay 900mhz transmitter, the sticks no longer move anything on the monitor. And it cannot be reflashed, and theres no RF output on either/or module. In or out.

After reboot, exactly the same. Only turning OFF both modules, allows the sticks and switches to work the various channels in the Monitor page. And reflashing the internal module is only possible if both are turned OFF and the radio restarted. Returning to the latest version on the FrSKY download page makes everything behave normally again.
Kind of serious!

This was a EU radio. It was corrected by using the Region Lua script. Which worked fine. As does the radio with the original firmware from the FrSKY download page. So... Now what?

Burgerman. burgerman@ntlworld.com

**Comment by ReinhardZ1** [comment]
> @ReinhardZ1
> Could you please confirm if the radio has the normal range(to confirm the output power is good)? We can not repeat it but I highly suspect there is something wrong with X9DP SE 2019 as I have several SWR reports of this radio as well.

Today I made another flight with my electric glider and this transmitter, but in ACCST D16 8CH mode. I think the radio has normal range. Flight distance was 250...300m

TX: X9DP SE2019, ISRM 2.1.0B, OTX2.3.7, LBT
RX: RX6R ACCST 2.1.0 LBT
External sensors: OXS Vario,GPS,VFAS;  Arduino LQBB5 sensor (value 5100...5104)

From time to time there is a spike in the telemetry data from ext. sensors (e.g. Vspd in the attached log), I have not observed that with ACCESS SW 2.1.0A on the receiver.

![Eclipse_200416_session4](https://user-images.githubusercontent.com/62476600/79476738-aafd2800-8009-11ea-86d5-e201ab87e926.jpg)
[Eclipse-2020-04-16-Session4.csv.txt](https://github.com/FrSkyRC/Firmware-Test/files/4488279/Eclipse-2020-04-16-Session4.csv.txt)

**Comment by Adelaxie** [comment]
@ReinhardZ1 300m is not a proof of normal range, could you test the range under range check mode? the range until the control is lost , of course I am not saying of real flight, just tests above the ground.

**Comment by ReinhardZ1** [comment]
> @ReinhardZ1 300m is not a proof of normal range, could you test the range under range check mode? the range until the control is lost , of course I am not saying of real flight, just tests above the ground.

I've made a range test with the X9DPSE2019 transmitter and also my old X9DP transmitter at the same location. Distances have been measured by GPS. See result in the pictures (the 5101 signal is from the link quality sensor and counts failsafe bits on SBus. The jump in the signal shows the end of the failsafe event)
![Eclipse_RangeTest_X9D+SE2019_RX6R_ACCST2 1 0](https://user-images.githubusercontent.com/62476600/79616852-54294880-8106-11ea-9e21-6903b3691f64.jpg)
![Eclipse_RangeTest_X9DP_RX6R_ACCST2 1 0](https://user-images.githubusercontent.com/62476600/79616863-5c818380-8106-11ea-8913-57c3f6e97e72.jpg)

**Comment by Adelaxie** [comment]
@ReinhardZ1 It seems its normal, but 15cm is not suggested btw. please keep the radio at least 1m above the ground.
Could you please do another test? (or just measure the output power under normal mode. if you have any devices to do so)
1. normal working mode, 
2. >1m above the ground, 
3. remove the receiver antenna to check the distance until signal lost,
4. do the same test for the good x9dp to compare.
Thanks in advance or you can send your radio to the nearest dealer with maintenance service to check the power output. I still suspect of the hardware issue.

**Comment by ReinhardZ1** [comment]
> @ReinhardZ1 It seems its normal, but 15cm is not suggested btw. please keep the radio at least 1m above the ground.
> Could you please do another test? (or just measure the output power under normal mode. if you have any devices to do so)
> 
> 1. normal working mode,
> 2. at least 1m above the ground,
> 3. remove the receiver antenna to check the distance until signal lost,
> 4. do the same test for the good x9dp to compare.
> 
> Thanks in advance or you can send your radio to the nearest dealer with maintenance service to check the power output. I still suspect of the hardware issue.

I will do no further testing or flying with this X9DPSE2019 transmitter. To be honest, I am very disappointed about the FrSky quality, if this is really a HW problem. The transmitter is only 6 month old and not often used.
I think here in germany Engel Modellbau is the right dealer with maintenance service. Maybe I will contact him.

**Comment by FlorianReck** [comment]
FrSky has the updates published. Still waiting for the X10 upgrade module, any estimated release for the test-firmware?

**Comment by TschorschR** [comment]
Only the ACCESS updates have been published (see https://www.frsky-rc.com/g-rx8/). I cant find the latest ACCST updates ;-(

Georg

**Comment by EngelMT** [comment]
Here it is. https://www.frsky-rc.com/g-rx8/

**Comment by TschorschR** [comment]
@Andreas: I can only find this version...

<img width="841" alt="Unbenannt" src="https://user-images.githubusercontent.com/60703945/79858399-a781ec00-83cf-11ea-9ff8-c211d940250b.PNG">

Which is dated 2020-03-24 aka 24th March

**Comment by franz4949** [comment]
Yeah, latest release version is E which was on 11th April, thus should read 2020-04-11

**Comment by EngelMT** [comment]
You are right. The Update time is not right. But Adela confirmed in the moment that the Update is the right one. Only the UpdateTime was not changed. So you can use it.

**Comment by TschorschR** [comment]
Guys, this is not serious! Not professional...
Imagine you would provide your services like that....

**Comment by Burgerman-X10express** [comment]
OK. You published the X10S Express update to the frSKY website.
If anyone downloads it and is using the FrOS Operating system that the radio sells with, like me, it freezes up! No module seen, no RF output, and no stick movement visible on the monitor field.

You told me to wait for FrOS update?

Surely there should either be an FrOS update up on your site too? 
Or wait, or at least a not to say it only works with OpenTX?

**Comment by Hennyvt** [comment]
Adela
I am using Horus X10S Express, found the following small issue. Opentx 2.3.7 ISRM-S-X10E_2.1.0.frsk
When flashing ACCESS firmware v2.1.0.frsk release version from FRsky download site to RX6R using my horus sometimes I receive a "Device not found" warning, if I immediately repeat the flashing process  most it is successful. Also using the STK module it does not support frsk file format and this needs to be entered manually every time it's used. Can frsk file format be added to the STK file formats. The STK windows interface looks very old school and probably needs a refresh.  

Henny

**Comment by ParkerEde** [comment]
@Adelaxie
Hi Adela,
Any ETA for the ISRM Horus X10 Upgrade Module?

Best regards,
Ralf

**Comment by brainbubblersbest** [comment]
Adela really?  Allmost any Information is better than no one.
Thats out of China a Mark for Service Quality. 
The will to communicate with customers. 
If you can just say it's in progress, or we had a few Problems to fix, before publish the bin.
Thats allmost better than the Sound of silence, after a request from a customer.

**Comment by Adelaxie** [comment]
it's in progress. :) and we just had a five days long holiday.

**Comment by ParkerEde** [comment]
Any ETA for the ISRM Horus X10 Upgrade Module?
one , two weeks, 1 month?

**Comment by ParkerEde** [comment]
I'm disappointed. Everyone can fly with the safe 2.1.0 version. GRX and SxR have some liitle bugs, but they can fly with 2.1.0 safe. Only the X10 ISRM Upgrade Module has no secure fix. Everyone who has this installed, can't fly safe. I think the ISRM Upgrade module should have the highest priority.
Kind regards
Ralf

**Comment by aalain8** [comment]
Good morning, 
Many of us have upgraded to an ACCESS upgrade module for the X10 because of the servo deflection problems I've experienced on several occasions.
I also think the V2.1.0 upgrade for the module is urgent.
It would also allow us to participate in the test. 
Having said that, thank you Adela for the work done and for the future.

**Comment by Adelaxie** [comment]
> Hi Adela, have you a modify Tool-FrSky update sport : .frsk files cannot be recognised

[frsky_update_sport_rev20.zip](https://github.com/FrSkyRC/Firmware-Test/files/4602798/frsky_update_sport_rev20.zip)

**Comment by Burgerman-X10express** [comment]
Recognised here.

**Comment by Burgerman-X10express** [comment]
Anyone know whats wrong with the SxR 2.1.0 updates?

**Comment by dennisr79** [comment]
> Just installed:
> ISRM-S-X10E for X10/X10S Express -
> FW-X10X10SEXPRESS-ISRM-v2.1.0J.zip
> FROS latest.
> On to a brand new and up to date, X10 Express. Twice in case  I something I did wrong or a corrupt file.
> It has the 900Mhz R9M 2019 module in the rear bay. On latest 1.3 Access  (NonEU).
> Provided both internal + external modules are turned OFF and the TX rebooted, the sticks then move the various channels on the Monitor screen.
> The moment the INTERNAL 2.4 ISRM is turned on, (or the Module bay 900mhz? not sure), the sticks no longer move anything on the monitor. And it cannot be reflashed, and theres no RF output on either/or module. Internal or external. And theres no RF out. (I have a signal strength meter). And the faint hissing on the speaker is gone.
> After reboot, exactly the same.
> Only turning OFF both modules, allows the sticks and switches to work the various channels in the Monitor page and only after another reboot. And reflashing the internal module is only possible if both modules are first turned OFF and the radio restarted. Returning it to the latest fw version from the FrSKY download page makes everything behave normally again.
> This was a EU radio. It was corrected to NonEU by using the Region.Lua script. Which worked fine. As does the radio with the original firmware from the FrSKY download page.
> So... Now what?
> Burgerman. burgerman@ntlworld.com
> EDIT:
> P.S. I have re-tested. Reloaded and tried again, - with same result.

same problem…
after update isrm 2.1.0.j  module was off……..no register, no binding , all receivers whit new access firmware
returned at older isrm version...all ok!

burgerman-x10express   do you have solution?

**Comment by EngelMT** [comment]
@dennisr79 Do you use the same configuration as burgerman use? And do you have the same issue? Do you use external RF too?

What is meant by FROS latest? Which version exactly?

**Comment by dennisr79** [comment]
Hello andreas,
Do you remember me? I' m the president of flying field GAL 2008 of LESTANS
(ITALY). 🙂
My radio is native x10s express whit fros 2.0.02 beta and no external
module.
Whit new irsm firmware the internal module was total off, no signal, no
register and binding whit receivers ( updated whit new firmwares).


Il lun 11 mag 2020, 18:31 Andreas Engel <notifications@github.com> ha
scritto:

**Comment by Burgerman-X10express** [comment]
The only solution was to install OpenTX since FrOS update seems to be missing in action. Then everything with the official 2.1.0 works. Exact same files as the j beta.

 

Not a thing I really wanted to do but its OK once you spend about a week figuring out how it works.

**Comment by EngelMT** [comment]
We have a beta test version from FROS here. It is the 2.0.06. This works fine. So next days I think Adela will offer this version for tests.

Hi Dennis, yes I remember you. Send you an email because it is not the place for write holiday or Airshow letters.LOL!!!

**Comment by Adelaxie** [comment]
Firmware for the X10S ISRM upgrade module is updated. Check the first post or CODE section to download.please test with opentx firmware with internalaccess option clicked when downloaded.
X12S canbe used with but need to take care of the wires connection and module installing.

**Comment by Burgerman-X10express** [comment]
Intend testing this S8R firmware on the bench with various sensors and a Horus express today.
Can anyone tell me what this is intended to fix over the previous 2.1.0 firmware that I had installed a week ago?  Thanks. 

In other words, what known issues did that previous version 1.2.0 version have that prevented it being posted on the official FrSKY download site along with all the other recievers?

**Comment by Burgerman-X10express** [comment]
Another question. Do all the R9M, R9 recievers etc also need to be updated soon to version 2.1.0 to fix the same problems?  Currently using these as redundancy. And it allows 2 seperate RSSI readings in telemetry. Which is great! 

Another question... What happens if one or the other, or both, recievers used in a redundant system are STABILITY ones. How does that work? Anyone know? Please tell me! So I can test a few things.

**Comment by gvt70** [comment]
@Adelaxie 

Please provide detailed installation manual and/or video

"X12S can be used with but need to take care of the wires connection and module installing. "

**Comment by Tiefflieger68** [comment]
there is an issue at the Upgrade ISRM Module for X10.

In case of disactivated, but mounted extenal antenna, I have an RSSI of 90db.
(distance 3m - RX6R with latest ACCESS FW)
Everything seems fine.
Now I unmount the external antenna and RSSI drops to 44db. Ups...

In case of activated external antenna, I have a RSSI of 90db. No matter if external antenna is mounted or not. 

2 ideas:
1: there is an issue at the ISRM firmware
2: the antenne ports of the module have changed. In this case you have to change the install video

This issue has been observed by a member of the German FrSky Forum and veryfied by me.

**Comment by Burgerman-X10express** [comment]
Another question.

In 2.1.0 for the internal module on the X10 Horus Express, I would like a definitive answer. When EXTERNAL antenna is selected, do the internal ones also operate? 

 

Thank you.

**Comment by hpolig** [comment]
Looks like they do not operate parallel to the external antenna.  #43

**Comment by Burgerman-X10express** [comment]
The post you linked to does not show that?  But it does show that the update module for the non express TX has a bug.

**Comment by Burgerman-X10express** [comment]
GVT70. To fit the X10 upgrade ISRM Access board, you will have to fit 2 internal antennas. The X12 only has 1 internal one. Those two are both ON when you choose "internal" and I THINK also when you choose external? 

The X10 board has 3 antenna connections. The two "inboard" ones are cycled alternately. Each is pointed in a different plane at 90 degrees. You will be able to use aN X10 or an X12 internal antenna. They are basically the same other than the cable length in the X10 one is much shorter. They are simple copper trace self adhesive things, around 3 dollars. The copper trace looks too short. At around 23mm from memory. But it reads the correct SWR ratio, 50Ohms once stuck to the plastic case. I measured them!

**Comment by Adelaxie** [comment]
antenna working mode was wrong. fixed, please continue the function tests.

**Comment by alm-nl** [comment]
> antenna working mode was wrong. fixed, please continue the function tests.

When I read your reply it was not clear a new version was posted even though you said fixed, but scrolling through the list I saw a new file dated 2020-05-18. So for those who had the same idea, check the list in the top. ;-)

**Comment by ParkerEde** [comment]
Antenna mode works now. While ext. antenna mode is active, I've unplugged external antenna. Now I got as expected the antenna defective warnung. Nice.
@Adelaxie Thank you for the fast response to this issue and best regards to you and your team.

**Comment by Adelaxie** [comment]
> > antenna working mode was wrong. fixed, please continue the function tests.
> 
> When I read your reply it was not clear a new version was posted even though you said fixed, but scrolling through the list I saw a new file dated 2020-05-18. So for those who had the same idea, check the list in the top. ;-)

Thanks, ， and always suggest checking the code section for updates and information/note.

**Comment by StefanKellerAC** [comment]
> Thanks, ， and always suggest checking the code section for updates and information/note.

also the latest betas for G-RX8 etc should be listed there.

**Comment by StefanKellerAC** [comment]
> ISRM-S-X10U for X10/X10S Upgrade Module with ISRM-S-X10S_2.1.0_200518Beta.zip

ACCST mode is not working, binding is not possible with RX4R or G-RX8

**Comment by MIKE-DAILEY** [comment]
Please post what is displayed on the "Modules / RX Version" page for the ISRM.  

Mike

**Comment by StefanKellerAC** [comment]
> ISRM-S-X10S
> 1.1.0/2.1.0FCC

I tested the RX4R with LBT and FCC Version, the G-RX8 only with LBT
( OTX 2.3.7 )

**Comment by MIKE-DAILEY** [comment]
If the ISRM is 2.1.0.FCC you need your ACCST receivers to use FCC firmware.  Did you wnt the ISRM to be FCC or EU (LBT)?

Mike

**Comment by hpolig** [comment]
There ist only one ISRM-S-X10S_2.1.0_200518Beta firmware for the ISRM and as far as i know it's only LBT, no FCC. I've successfully bound a S8R and a X8R with ACCST 2.1.0 LBT .

**Comment by MIKE-DAILEY** [comment]
But your ISRM is in FCC mode now and will only work with ACCST D16 receivers that are flshed with FCC firmware.

**Comment by hpolig** [comment]
Double checked it, i am using only LBT firmware for the mentioned receivers and the 200518 beta for the ISRM in my X10S and it works

**Comment by MIKE-DAILEY** [comment]
But you posted this:

ISRM-S-X10S
1.1.0/2.1.0FCC

Was it actually: 

ISRM-S-X10S
1.1.0/2.1.0EU

**Comment by hpolig** [comment]
Hold on, I don't have a problem, it's Stefan who has the problem.

Is it possible that the update takes over the latest version (LBT/FCC) of the previous firmware?

**Comment by MIKE-DAILEY** [comment]
Missed that.  Thanks.  The ISRM should stay in its FCC or EU mode and not change with a firmware mode, but things could go wrong.  

Mike

**Comment by StefanKellerAC** [comment]
I need LBT 
today I tested again with FCC:
Rx4R with FCC does **not** bind, going away to bind also does **not** work
G-RX8 binds !
(of course I would use ACCESS with those reicevers)

**Comment by StefanKellerAC** [comment]
I need LBT and would use ACCESS with those receivers, but:
today I tested with FCC 2.1:
Rx4R and G-RX8 does bind !

**Comment by Joetrin** [comment]
ISRM v 2.1.0 in Xlite Pro, for some reason it flashes the radio in EU mode.  It will not bind wiht RX6R 2.1.0 ACCST or ACCESS..

**Comment by IronFenyx** [comment]
Hi! I have a problem. After update my taranis Q X7 ACCESS to v 2.1.0 i cannot bind to xm+ reciever!
Modules / RX Version
Int. Module
Module ISRM-N
1.1.0 / 2.1.0**EU**
How can i change it to **US**???

**Comment by Hennyvt** [comment]
Did you upgrade the XM+ to v2.1.0 ???

**Comment by Adelaxie** [comment]
> @Adelaxie
> ISRM v 2.1.0 in Xlite Pro, for some reason it flashes the radio in EU mode. It will not bind wiht RX6R 2.1.0 ACCST or ACCESS..
> 
> Prior to the reflash, this unit was running in FCC mode.
> 
> Firmware that was used is the released version: https://www.frsky-rc.com/wp-content/uploads/Downloads/Firmware/ACCESS-2.x.x/FW-XLiteSPro-ISRM-v2.1.0.zip
> 
> Note: Inside the Zip file there is only 1 version for the XLite-Pro, that's the one that I used to update this TX. Is there a way to change it back to FCC or NonEU?
> 
> @MIKE-DAILEY / @Adelaxie are you aware if there is a new ISRM/ACCESS
> When I flashed my Xlite-Pro it changed the FCC to EU
> 
> Modules / RX Version
> Int. Module
> Module ISRM-Pro, 1.1.0 / 2.1.0EU
> 
> Is there a new firmware that addresses this, please?
> 
> Thank You

I have shared the lua tool through skype please for your own use.

**Comment by TschorschR** [comment]
Hi there. 
I just bought a brand new X9D + 2019.

It is soooo frustrating. I used several SD cards (even the one from my older X9D) and none of them is recognized. How can that be that such a product is on the market? It makes me really mad to waste my time playing around. I am aware that it is not a Jeti or Futaba but can I not expect basic stuff to work? 

It's really not good. I hope Frsky is reading this!

Georg

**Comment by Joetrin** [comment]
> Hi there.
> I just bought a brand new X9D + 2019.
> 
> It is soooo frustrating. I used several SD cards (even the one from my older X9D) and none of them is recognized. How can that be that such a product is on the market? It makes me really mad to waste my time playing around. I am aware that it is not a Jeti or Futaba but can I not expect basic stuff to work?
> 
> It's really not good. I hope Frsky is reading this!
> 
> Georg

Hi Georg,

Firstly, please format your SD card before using, then copy the correct files for the version of OpenTX you are using, and go from there.

Thank you

**Comment by TschorschR** [comment]
All done of course!

SD card is recognized by Windows with adapter  it not in Taranis so its not the format

**Comment by Joetrin** [comment]
> All done of course!
> 
> SD card is recognized by Windows with adapter but not in Taranis with USB cable (and yes, I tried different cables) so its not the format. And yes, I downloaden the right SD content.

What size is the SD card, please?

**Comment by TschorschR** [comment]
16, 32, 64gb

And....why do i need to unplug the battery to get into bootload again? And potentially breaking the connector?

**Comment by Joetrin** [comment]
> 16, 32, 64gb
> 
> And....why do i need to unplug the battery to get into bootload again? And potentially breaking the connector?

You may have a faulty unit, please contact the people you purchased your radio from and have it repaired or replaced.

**Comment by TschorschR** [comment]
That's not a good product......

**Comment by FlorianReck** [comment]
@Adelaxie  please moderate here, that's not stuff to be discussed Herr. 

@TschorschR: Return your unit as advised by @Joetrin. Thanks.

**Comment by TschorschR** [comment]
So, I know I should return, but here's another example.

The readme.txt of G-RX8 points you to "https://www.frsky-rc.com/how-to-use-the-transmitter-to-flash-the-firmware-of-the-**x8r**-receiver/" how to flash G-RX8. How  can that be? X8R is sooo way different to  G-RX8! And of course, flashing  G-RX8 does not work according to that manual.

And yes, this is the right place to drop these type of msgs because Frsky is not up to the game, but  once you have 10+ receivers you are kind of locked and that sucks!

Btw, flashing FW via OpenTx does not work, neither. I had to copy the 2.3.7 file onto the SD card and flash it via the transmitter and not via OpenTx. Shame shame....

G.

**Comment by Hennyvt** [comment]
OK you are missing the point here flashing the X8R is only an example of what's involved. The S-port connector on the G-RX8 is on the small white connector on top of the G-RX8. Check the manual that came with the receiver. I personally would not remove the SD card everytime from the back of the radio. Simply plug a USB cable into the radio and copy the file to the radios SD card.
Now depending which version of firmware you have on your radio (v2.1.0 release version) you must also flash the receiver with the same version. Its available on the FRsky web site. I have flashed my G-RX8 both with ACCST and ACCESS and they both work fine. If you are in the EU you must choose the EU version of the firmware.

**Comment by IronFenyx** [comment]
> 
> 
> Did you upgrade the XM+ to v2.1.0 ???

Yes.
The problem was that the Chinese from aliexpress sent me the wrong version of the equipment! I ordered the FCC, and they sent me the EU! Rogues! And it became clear only now. After flashing XM+ to the EU version, the binding is completed. Is there a way to change the firmware to the FCC?

**Comment by Burgerman-X10express** [comment]
Yes. You just flash the XM+ to the EU or the NEU (FCC) version as you wish. Or do you mean swap the TX to NEU or EU?  And again thats easy to do as well... With an Region.lua script.
So yes, both ways, RX or TX can be either...

**Comment by Hennyvt** [comment]
This lua script is only relevant for transmitters using the ISRM RF module such as Horus Express etc.
Here is a link to the script.
https://blog.seidel-philipp.de/change-frsky-isrm-mode-eu-lbt-fcc-via-lua-script/#LUA_Script_for_FrSky_ISRM_Module

**Comment by Burgerman-X10express** [comment]
Thats not correct. Theres one that was published for the X10 without the ISRM probably in error. It is in the official download for the transmitter for fros. It will require you to replace FrOs temporarily. I also have one that works in OpenTX I think, but untested by me.

**Comment by Tiefflieger68** [comment]
will there be an bug fix for the G-RX8 ACCESS regarding "Optimized the vario sensor output" as like for the ACCST firmware ?

**Comment by IronFenyx** [comment]
> Yes. The hardware is identical. You just flash the XM+ to the EU or the NEU (FCC) version as you wish. Or do you mean swap the TX to NEU or EU? And again thats easy to do as well... With an Region.lua script.
> So yes, both ways, RX or TX can be either...

I need to swap the TX to **FCC** version. Where can i find the Region.lua  script for my x7?

**Comment by Hennyvt** [comment]
Yes this is correct.
https://blog.seidel-philipp.de/change-frsky-isrm-mode-eu-lbt-fcc-via-lua-script/#LUA_Script_for_FrSky_ISRM_Module

**Comment by StefanKellerAC** [comment]
@Adelaxie which receivers will get a F.Port 2.0 update, to use with the new servos?

**Comment by cnagayama** [comment]
Is there a version for the x-lite or is it not needed?  I am unable to bind r-xsr with most recent firmware, but can use the one just before it.  However, it does not utilize the correct rssi and says telemetry lost at 85 db.  Rx that have not been flashed with newer firmware work just fine (bought and installed a brand new r-xsr without updating firmware to verify).

**Comment by cnagayama** [comment]
Does anybody have the original firmware that comes on a stock r-xsr rx?  I'm just trying to avoid changing my x-lite because everything else works fine right now...

**Comment by franz4949** [comment]
It's still on the FrSky R-XSR download page. Look at ACCST firmware and there you find
 "History firmware **DOWNLOAD** (not compatible with v2.x.x)"

**Comment by cnagayama** [comment]
Tried that one...I lose telemetry around 80 to 85db but still have full control.

**Comment by ignaciobfp** [comment]
> 
> 
> Thats not correct. Theres one that was published for the X10 without the ISRM probably in error. It is in the official download for the transmitter for fros firmware. Along with the lua scripts. It will require you to replace FrOs temporarily. I also have one thats modified and works in OpenTX I think, but untested by me.
> 
> Edit. Just checked, and its now gone from the x10 firmware folder and theres now many different versions there. It was there for at least 8 months. I actually downloaded that Region.lua file for the x10 and used it successfully on my Express version. So obviously the same file can do both. If you want a copy just ask.

Hello. I got sent the wrong ISRM upgrade kit and need the script aswell. Could yo please send me the script for the x10 please? I do not have another radio to use the other version.

**Comment by Hennyvt** [comment]
What is it that you actually want. I now have the Horus X10S Express. Do you want the IRSM for the X10S non Express. I only have an early version, for some reason FRsky never publish a final release version.

**Comment by ignaciobfp** [comment]
> 
> 
> What is it that you actually want. I now have the Horus X10S Express. Do you want the IRSM for the X10S non Express. I only have an early version, for some reason FRsky never publish a final release version.

I currently have a X12S in which I recently replaced internal module with IRSM upgrade kit. I live in europe but the shop (in europe aswell) sent me the FCC version of the module (for what I read, they are the only ones available at the moment).

I flashed my receivers with 2.1 FCC firmware and they work as expected, but the law here requires LBT on all devices, and I would prefer to do so, but the available script for region change only works on radios with monochrome screens like taranis. I hoped the script for the X10, even from an early version, would work on my radio.

**Comment by Burgerman-X10express** [comment]
Email me burgerman@ntlworld.com <mailto:burgerman@ntlworld.com>  and I will send…

**Comment by GlidLov** [comment]
i think i found a quite big bug in 2.1.0 when used on a X8R with latest 2.1.0 fw and QX7S ACCESS with ISRM 2.1.0 on quadcopters with Pixhawk and Arducopter 4.0:
 They bind correctly, also signal is correctly read by pixhawk, then when you try to arm motor, pixhawk loose SBUS decoding and act like it loose signal from TX. Then, as soon as it disarms due to timeout (auto disarm), it regains from failsafe.
If i flashback TX to ISRM 1.1.1 / 1.1.3 and X8R to old LBT ACCST fw, it behaves correctly and Arms motor without problems.
I also tried with 2.1.0 with newer "archer" series and it seems to be ok either, but i only did a couple of bench tests.

**Comment by Hennyvt** [comment]
Sounds like FS flag is not correctly set on the X8R

**Comment by brainbubblersbest** [comment]
> 1. ISRM-N firmware for **Q X7/X7S ACCESS, X9 Lite**
>    [FW-QX7X7SACCESS-X9Lite-ISRM-v2.1.0J.zip](https://github.com/FrSkyRC/Firmware-Test/files/4478726/FW-QX7X7SACCESS-X9Lite-ISRM-v2.1.0J.zip)
> 2. ISRM-S-X9LiteS for **X9 Lite S**
>    [FW-X9LiteS-ISRM_v2.1.0J.zip](https://github.com/FrSkyRC/Firmware-Test/files/4478728/FW-X9LiteS-ISRM_v2.1.0J.zip)
> 3. ISRM-S-X10E for **X10/X10S Express**
>    [FW-X10X10SEXPRESS-ISRM-v2.1.0J.zip](https://github.com/FrSkyRC/Firmware-Test/files/4478730/FW-X10X10SEXPRESS-ISRM-v2.1.0J.zip)
> 4. ISRM-S-X10U for **X10/X10S Upgrade Module**
>    [ISRM-S-X10S_2.1.0_200518Beta.zip](https://github.com/FrSkyRC/Firmware-Test/files/4641736/ISRM-S-X10S_2.1.0_200518Beta.zip)

Sorry, but this is not better to describe than a bad joke. 
Already the users have mentioned to frsky. The Staff should keep better notice of it's Version numbering.
Can you please explain to us all how this can be 100% sort out?
You publish a FW  for:
ISRM-S-X10U for **X10/X10S Upgrade Module**
And then the File is Named as:
[**ISRM-S-X10S**_2.1.0_200518Beta

It seems sometimes that the biggest problem at FRSKY is to work with basic accurateness. How should this way to Do your stuff, not leading to more Problems in Product Designing.
If you would Do your Basics as it should be, then Frsky had no reason to fear from the other New RC Companys. 
Thats how every European or US See it. But in Chinese this seems not even imaginable.
